pkg/resources/aws/routetable.go

```diff
@@ -51,7 +51,7 @@ func DescribeRouteTables(cloud fi.Cloud, clusterName string) (map[string]*ec2.Ro
 	return routeTables, nil
 }
 
-func ListRouteTables(cloud fi.Cloud, clusterName string) ([]*resources.Resource, error) {
+func ListRouteTables(cloud fi.Cloud, vpcID, clusterName string) ([]*resources.Resource, error) {
```

Comment:
I see that many of the functions are now taking vpc-id which is nice, not just  in the `eni.go` file but also in  other files like sqs, routetable etc. What I'm trying to understand is, are we not using these functions anywhere else in the project ? Because I don't see any consumers of these functions are being changed to accommodate this change i.e; the callers ?

Reply:
These are used only on cleanup, which is quite demanding.
`vpc-id` can be used as a filter later in other places. For now, let's see how the review goes.